test/python/fx_importer/sparse_test.py

```diff
@@ -567,9 +575,8 @@ def forward(self, X):
                 mem = mem * (1.0 - spike)
                 spike_pot.append(spike)
             spike_pot = torch.stack(spike_pot, dim=-1)
-            # TODO: we would like to see something like
-            #       return spike_pot.to_sparse()
-            return spike_pot
+            # Hack to get to_sparse() through propagation.
+            return spike_pot.to_sparse().to_dense()
```

Comment:
can we put this before `torch.stack`?

Reply:
Oh, sure, I keep forgetting what way is best. Let me change.